homeassistant/components/trafikverket_ferry/sensor.py

```diff
@@ -131,7 +132,7 @@ def __init__(
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
-        self._attr_name = f"{name} {entity_description.name}"
+        self._attr_name = entity_description.name
```

Comment:
Entities, by default, pick up the name from the entity description (when the entity description is set), which means this line isn't needed.

```suggestion
```